x-pack/legacy/plugins/maps/public/layers/sources/es_search_source/es_search_source.js

```diff
@@ -56,9 +56,10 @@ export class ESSearchSource extends AbstractESSource {
       geoField: descriptor.geoField,
       filterByMapBounds: _.get(descriptor, 'filterByMapBounds', DEFAULT_FILTER_BY_MAP_BOUNDS),
       tooltipProperties: _.get(descriptor, 'tooltipProperties', []),
+      sortField: _.get(descriptor, 'sortField', ''),
+      sortOrder: _.get(descriptor, 'sortOrder', SORT_ORDER.DESC),
```

Comment:
consider wrapping this in single config object
```
sort: {
 order: ...,
 field: ...
}
```

looks a little cleaner, as they are co-dependent